app/services/dashboard/loader.rb

```diff
@@ -6,7 +6,7 @@ module Loader
                        {:template=>'reports_widget',      :sizex=>6,:sizey=>1,:name=> N_('Report summary')},
                        {:template=>'distribution_widget', :sizex=>6,:sizey=>1,:name=> N_('Distribution chart')}]
     # Widget templates that are allowed on dashboard. Default widgets automatically allow their templates.
-    ALLOWED_TEMPLATES = []
+    ALLOWED_TEMPLATES = ['trends/trend_chart']
```

Comment:
I am a bit concerned about the `ALLOWED_TEMPLATES`.
Can you describe a use case when the template will not be allowed, but it will be specified in the widget?